external/fv3kube/fv3kube/base_yamls/SHiELD/PIRE-like/fv3config.yml

```diff
@@ -0,0 +1,295 @@
+    z_slope_ice: true
+    z_slope_liq: true
+  gfs_physics_nml:
+    cal_pre: false
+    cap_k0_land: false
+    cdmbgwd:
+    - 3.5
+    - 0.25
+    cloud_gfdl: true
+    cnvcld: false
+    cnvgwd: true
+    debug: false
+    do_deep: true
+    do_ocean: false
+    dspheat: true
+    fhcyc: 24.0
+    fhlwr: 1800.0
+    fhswr: 1800.0
+    fhzero: 3.0
+    gwd_p_crit: 2000.0
+    hybedmf: false
+    iaer: 111
+    ialb: 1
+    ico2: 2
+    iems: 1
+    imfdeepcnv: 2
+    imfshalcnv: 2
+    isatmedmf: 1
+    isol: 2
+    isot: 1
+    isubc_lw: 2
+    isubc_sw: 2
+    ivegsrc: 1
+    ldiag3d: true
+    lwhtr: true
+    ncld: 5
+    nst_anl: true
+    pdfcld: true
+    pre_rad: false
+    prslrd0: 0.0
+    random_clds: false
+    redrag: true
+    rlmx: 500.0  # Restore default from PIRE simulation codebase (current default is 300.0)
+    satmedmf: true
+    shal_cnv: true
+    swhtr: true
+    trans_trac: true
+    use_ext_sst: false  # Previously use_ec_sst
+    use_ufo: true
+    xkzm_hi: 1.5
+    xkzm_hl: 2.0
+    xkzm_mi: 1.5
+    xkzm_ml: 2.0
+    xkzminv: 0.0
+    ysupbl: false
+    zhao_mic: false
+  integ_phys_nml:
+    do_inline_mp: true  # Moved from fv_core_nml
+    do_sat_adj: false  # Moved from fv_core_nml
+  namsfc:
+    fabsl: 0  # Use maximum snow albedo from forcing files instead of initial conditions
+    fabss: 0  # Use maximum snow albedo from forcing files instead of initial conditions
+    faisl: 0  # Use land / sea / sea-ice mask from forcing files instead of initial conditions
+    faiss: 0  # Use land / sea / sea-ice mask from forcing files instead of initial conditions
+    fnabsc: INPUT/fix_sfc/maximum_snow_albedo.tileX.nc
+    fnacna: ''
+    fnaisc: grb/CFSR.SEAICE.1982.2012.monthly.clim.grb
+    fnalbc: INPUT/fix_sfc/snowfree_albedo.tileX.nc
+    fnalbc2: INPUT/fix_sfc/facsf.tileX.nc
+    fnglac: grb/global_glacier.2x2.grb
+    fnmskh: grb/seaice_newland.grb
+    fnmxic: grb/global_maxice.2x2.grb
+    fnslpc: INPUT/fix_sfc/slope_type.tileX.nc
+    fnsmcc: grb/global_soilmgldas.t1534.3072.1536.grb
+    fnsnoa: ''
+    fnsnoc: grb/global_snoclim.1.875.grb
+    fnsotc: INPUT/fix_sfc/soil_type.tileX.nc
+    fntg3c: INPUT/fix_sfc/substrate_temperature.tileX.nc
+    fntsfa: ''
+    fntsfc: grb/RTGSST.1982.2012.monthly.clim.grb
+    fnvegc: INPUT/fix_sfc/vegetation_greenness.tileX.nc
+    fnvetc: INPUT/fix_sfc/vegetation_type.tileX.nc
+    fnvmnc: INPUT/fix_sfc/vegetation_greenness.tileX.nc
+    fnvmxc: INPUT/fix_sfc/vegetation_greenness.tileX.nc
+    fnzorc: igbp
+    fsicl: 0  # Use sea ice fraction from forcing files instead of persisting the fraction in the initial condition
+    fsics: 0  # Use sea ice fraction from forcing files instead of persisting the fraction in the initial condition
+    fslpl: 99999  # Use slope type from initial condition
+    fsmcl:
+    - 99999  # Use soil moisture from initial condition
+    - 99999
+    - 99999
+    - 99999
+    fsnol: 99999  # Use snow cover fraction from initial condition
```

Comment:
Recalling from the radiation-cloud ML project, the snow cover fraction resulting from coarsened restart initialization was very high for the first few timesteps before the snow/land surface model was able to melt it away to something more reasonable ([notebook](https://github.com/ai2cm/explore/blob/master/brianh/2023-06-07-fv3gfs-SHiELD-radiation-differences/2023-06-07-fv3gfs-vs-PIRE-like-albedo-differences.ipynb)). I ended up switching to climatological snow covered area because for a shorter run that initial shock was still significant over the run. Is there a use case for a run with the snow covered area held to these initial conditions?

Reply:
Thanks for reminding me about the initialization issue when running with coarse restarts.  Indeed the focus of this config is longer runs, specifically new reference simulations—typically started from more conventional initial conditions—so I tentatively prefer to keep things this way in the base config.